Index: www/extras/editarea/edit_area/langs/cs.js
===================================================================
--- www/extras/editarea/edit_area/langs/cs.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/cs.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/mk.js
===================================================================
--- www/extras/editarea/edit_area/langs/mk.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/mk.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/fr.js
===================================================================
--- www/extras/editarea/edit_area/langs/fr.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/fr.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/es.js
===================================================================
--- www/extras/editarea/edit_area/langs/es.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/es.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_coldfusion: "Coldfusion",
Index: www/extras/editarea/edit_area/langs/nl.js
===================================================================
--- www/extras/editarea/edit_area/langs/nl.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/nl.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/hr.js
===================================================================
--- www/extras/editarea/edit_area/langs/hr.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/hr.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/pl.js
===================================================================
--- www/extras/editarea/edit_area/langs/pl.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/pl.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/it.js
===================================================================
--- www/extras/editarea/edit_area/langs/it.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/it.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/sk.js
===================================================================
--- www/extras/editarea/edit_area/langs/sk.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/sk.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/zh.js
===================================================================
--- www/extras/editarea/edit_area/langs/zh.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/zh.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/pt.js
===================================================================
--- www/extras/editarea/edit_area/langs/pt.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/pt.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/ru.js
===================================================================
--- www/extras/editarea/edit_area/langs/ru.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/ru.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/de.js
===================================================================
--- www/extras/editarea/edit_area/langs/de.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/de.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/bg.js
===================================================================
--- www/extras/editarea/edit_area/langs/bg.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/bg.js	(working copy)
@@ -61,7 +61,7 @@
 syntax_cpp: "C++",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/ja.js
===================================================================
--- www/extras/editarea/edit_area/langs/ja.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/ja.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/dk.js
===================================================================
--- www/extras/editarea/edit_area/langs/dk.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/dk.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/fi.js
===================================================================
--- www/extras/editarea/edit_area/langs/fi.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/fi.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
@@ -64,4 +64,4 @@
 syntax_coldfusion: "Coldfusion",
 syntax_java: "Java",
 close_tab: "Sulje tiedosto"
-};
\ No newline at end of file
+};
Index: www/extras/editarea/edit_area/langs/en.js
===================================================================
--- www/extras/editarea/edit_area/langs/en.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/en.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
Index: www/extras/editarea/edit_area/langs/eo.js
===================================================================
--- www/extras/editarea/edit_area/langs/eo.js	(revision 10650)
+++ www/extras/editarea/edit_area/langs/eo.js	(working copy)
@@ -55,7 +55,7 @@
 syntax_cpp: "CPP",
 syntax_basic: "Basic",
 syntax_pas: "Pascal",
-syntax_brainfuck: "Brainfuck",
+syntax_brainfuck: "Bf",
 syntax_sql: "SQL",
 syntax_ruby: "Ruby",
 syntax_robotstxt: "Robots txt",
@@ -64,4 +64,4 @@
 syntax_coldfusion: "Coldfusion",
 syntax_java: "Java",
 close_tab: "Fermi la dosieron"
-};
\ No newline at end of file
+};
